tests/utils/fs/index.js

```diff
@@ -17,6 +18,12 @@ function getTmpFilePath(fileName) {
   return path.join(getTmpDirPath(), fileName);
 }
 
+function createFile(name, content) {
```

Comment:
It might be more to the point to name it `createTmpFile` (as we're taking relative path which we'll join with temp dir path)

Reply:
Yes, that makes sense. I thought it's "obvious" because it's in the `test` utils codebase but I agree, that making it more explicit is a good idea :+1: